backend/src/user/user.controller.ts

```diff
@@ -121,15 +136,17 @@ export class UserController {
    * @returns deleted user object
    */
   @Delete(":id")
-  @ApiOperation({ summary: "Delete data of specified user" })
-  @ApiOkResponse({ description: "The resource was returned successfully" })
+  @ApiOperation({ summary: API_OPERATIONS.DELETE_USER_SUMMARY })
+  @ApiOkResponse({ 
+    description: API_RESPONSES_DESCRIPTION.SUCCESSFUL_UPDATE_USER_INFORMATION_DESCRIPTION 
+  })
   @ApiForbiddenResponse({
     description:
-      "Unauthorized Request: Client does not have access rights to the requested content",
+    API_RESPONSES_DESCRIPTION.UNAUTHORIZED_ACCESS_DESCRIPTION,
   })
-  @ApiNotFoundResponse({ description: "Not Found: Resource not found" })
+  @ApiNotFoundResponse({ description: API_RESPONSES_DESCRIPTION.NOT_FOUND_DESCRIPTION })
   @ApiBadRequestResponse({
-    description: "Bad Request: id specified is invalid",
+    description: API_RESPONSES_DESCRIPTION.BAD_REQUEST_INVALID_CREDENTIALS_DESCRIPTION,
```

Comment:
not sure if invalid credentials is the correct error here? shld be an invalid ID right